src/shave.js

```diff
@@ -55,11 +55,13 @@ export default function shave(target, maxHeight, opts) {
   }
 }
 
-const plugin = window.$ || window.jQuery || window.Zepto;
-if (plugin) {
-  plugin.fn.extend({
-    shave: function shaveFunc(maxHeight, opts) {
-      return shave(this, maxHeight, opts);
-    },
-  });
+if (typeof window !== 'undefined') {
```

Comment:
@yowainwright I think you're right on: condensing the expression into one line could save a couple bytes, at the expense of making the code slightly harder to read. One tiny advantage you get from the nested version is that the inner `if (plugin)` expression can get skipped completely if there is no `window` object, which could make it a hair faster. It actually looks like there's actually only 1 byte difference in the size of the minified file, so we're really just splitting hairs - I'll leave it up to you. Thanks for looking at this so quickly!